app/controllers/application_controller.rb

```diff
@@ -1,5 +1,20 @@
 class ApplicationController < ActionController::Base
```

Comment:
its in the application controller bc it goes on every page. if we want a photo on a specific page we use that controller(item photo, put in item controller)